no_graph_implementation.py

```diff
@@ -0,0 +1,42 @@
+import doctest
+
+INPUT_FILE = open("career.in", "r")
+OUTPUT_FILE = open("career.out", "w")
+
+
+def get_input():
+    """
+    Function to get list of lists of vertexes
+    :return: list of levels
+    """
+    result = []
+    income = INPUT_FILE.readlines()
+    for i in range(len(income)):
```

Comment:
Why not just `for s in INPUT_FILE.readlines()` ? Or even a `map` instead of cycle? (I guess all you need is string.strip)